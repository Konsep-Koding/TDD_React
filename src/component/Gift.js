import React, { Component } from 'react'

 class Gift extends Component {
    constructor(props) {
        super(props);
        this.state = {
            person: '',
            present: ''
        };
      }
    render() {
        return (
          <div></div>
        )
    }
}

export default Gift
