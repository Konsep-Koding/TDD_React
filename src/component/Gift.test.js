import React  from 'react'
import Enzyme,{shallow, mount, render, configure} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Gift from './Gift'

Enzyme.configure({ adapter: new Adapter() });


describe('Gift', () => {
 const gift = shallow(<Gift/>)
 it('should render screen', () => {
     expect(gift).toMatchSnapshot();
 });

 it('should initial person in `state`', () => {
    expect(gift.state()).toEqual({ person: '', present: '' });
 });
 
});

