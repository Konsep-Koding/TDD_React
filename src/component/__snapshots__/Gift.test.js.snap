// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Gift should render screen 1`] = `ShallowWrapper {}`;
